kalite/legacy/updates_settings.py

```diff
@@ -66,7 +66,15 @@
 BACKUP_VIDEO_SOURCE = getattr(local_settings, "BACKUP_VIDEO_SOURCE", None)
 BACKUP_THUMBNAIL_SOURCE = getattr(local_settings, "BACKUP_THUMBNAIL_SOURCE", None)
 
-UPDATES_DATA_PATH = os.path.join(os.path.dirname(__file__), "data")
+# This is the standard method... but doesn't work because we cannot load
+# kalite.i18n while loading settings because of its __init__.py
+# from pkgutil import get_data
+# I18N_DATA_PATH = get_data("kalite.i18n", "data")
+
+# Use resource_filename instead of get_data because it does not try to open
+# a file and does not complain that its a directory
+from pkg_resources import resource_filename
+UPDATES_DATA_PATH = resource_filename("kalite", "updates/data")
```

Comment:
Never use `os.path`, if you need data files, let pkgutils find them for you.

`get_data` would be conventional, but the problem is that it resolves module paths using import mechanisms, and because of circularity, we cannot resolve a lot of imports in the the `settings` package.. so the commented out patter might be the future.

Overall, though: We should always set paths in settings modules, and not leave them spread all over the codebase! :)